src/modules/feature-modules/innovator/pages/innovation/record/section-edit.component.ts

```diff
@@ -26,204 +27,218 @@ export class InnovationSectionEditComponent extends CoreComponent implements OnI
-          // If more exceptions appears, a wizard configurations should be considered.
-          if (this.sectionId === 'INNOVATION_DESCRIPTION' && this.wizard.currentStepId === 1) {
-            this.stores.context.updateInnovation({ name: this.wizard.getAnswers().name });
-          }
-          return of(true);
-        }),
-        concatMap(() => {
-          const shouldRefreshInformation = this.wizard.currentStep().saveStrategy === 'updateAndWait';
-
-          if (shouldRefreshInformation) {
-            return this.stores.innovation.getSectionInfo$(this.innovation.id, this.sectionId);
-          } else {
-            return of({ data: {} });
-          }
-        })
-      )
-      .subscribe({
-        next: response => {
-          // Update only if GET call was made!
-          if (Object.keys(response.data).length > 0) {
-            this.wizard.setAnswers(this.wizard.runInboundParsing(response.data)).runRules();
-          }
-
-          this.wizard.nextStep();
-
-          if (this.wizard.isQuestionStep()) {
-            this.setPageTitle(this.wizard.currentStepTitle(), { showPage: false });
-          } else {
-            this.setPageStatus('LOADING');
-
-            this.stores.innovation.getSectionInfo$(this.innovation.id, this.sectionId).subscribe(sectionInfo => {
-              const validInformation = this.wizard.validateData();
-
-              if (!validInformation.valid) {
-                this.alertErrorsList = validInformation.errors;
-                this.setAlertError(`Please verify what's missing with your answers`, {
-                  itemsList: this.alertErrorsList,
-                  width: '2.thirds'
-                });
-              }
+    let currentStepIndex = this.wizard.currentStepId;
 
-              if (sectionInfo.status === 'DRAFT') {
-                this.submitButton.isActive = validInformation.valid;
-                if (
-                  this.innovation.status !== InnovationStatusEnum.CREATED &&
-                  this.innovation.status !== InnovationStatusEnum.WAITING_NEEDS_ASSESSMENT
-                ) {
-                  this.submitButton.label = !this.isArchived ? 'Submit updates' : 'Save updates';
-                }
-              }
+    if (typeof currentStepIndex === 'number') {
+      if (action === 'previous') {
+        const previousStep = this.wizard.getPreviousStep(this.isChangeMode);
+        if (this.wizard.isFirstStep() || previousStep === -1) {
+          this.redirectTo(this.baseUrl);
+        } else {
+          this.onGoToStep(previousStep, this.isChangeMode);
+        }
+      }
 
-              for (const [index, item] of this.wizard.getSummary().entries()) {
-                this.displayChangeButtonList.push(index);
-                if (!item.value && !item.isNotMandatory) {
-                  break;
-                }
+      if (action === 'next') {
+        const shouldUpdateInformation =
```

Comment:
Not really, this helps with avoiding making a call to update when the answer is the same. In the case of fields-group, it will always update, regarding of the answer (it was already working this way in the current wizard)